lightbulb/context/base.py

```diff
@@ -529,7 +532,8 @@ async def _editor(
         proxy = ResponseProxy(
             fetcher=self._interaction.fetch_initial_response,
             editor=functools.partial(_editor, inter=self._interaction)
-            if hikari.MessageFlag.EPHEMERAL in kwargs.get("flags", hikari.MessageFlag.NONE)
+            if hikari.MessageFlag.EPHEMERAL & kwargs.get("flags", hikari.MessageFlag.NONE)
+            == hikari.MessageFlag.EPHEMERAL
```

Comment:
This is used twice so might make sense to create a helper lambda or similar
```py
includes_ephemeral = lambda flags: (hikari.MessageFlag.EPHEMERAL & flags) == hikari.MessageFlag.EPHEMERAL
```
for example
```py
if includes_ephemeral(kwargs.get("flags", hikari.MessageFlag.NONE)):
    ...
```
etc

Reply:
ok where should I put the lambda

Reply:
prob just at the top of the method
doesn't need to be available to any other functions

Reply:
check latest commit, is this ok?